gatekeeper-service/pkg/handler/approval_finished_event_handler.go

```diff
@@ -93,6 +93,10 @@ func (a *ApprovalFinishedEventHandler) handleApprovalFinishedEvent(inputEvent ke
 		} else {
 			a.keptn.Logger.Info(fmt.Sprintf("Rejection for image %s for service %s of project %s and current stage %s received",
 				inputEvent.Image, inputEvent.Service, inputEvent.Project, inputEvent.Stage))
+			if err := closeOpenApproval(inputEvent, triggeredID); err != nil {
```

Comment:
The code is duplicated from line 89-92 (if clause) to line 96-99 (else clause).

Any objections to move it below the if-else block so that it is executed in both cases? Or is there a case I am missing